architecture/additional_concepts/storage.adoc

```diff
@@ -496,11 +496,15 @@ spec:
 [[pvc-storage-class]]
 === Storage Class
 
-Claims can optionally request a specific `StorageClass`. Dynamic provisioners
-are configured by the cluster administrator to service one or more storage
-classes. They create a PV on demand that matches the specifications in the PVC,
-if they are able. It is also possible for the cluster administrator to set a
-default `StorageClass` for all PVCs.
+Claims can optionally request a specific `StorageClass` by specifying its name
+in the `storageClassName` attribute. Only PVs of the requested class, ones with the same `storageClassName` as the PVC, can be bound to the PVC. Dynamic provisioners are configured by the
+cluster administrator to service one or more storage classes. They create a PV
```

Comment:
Active: "The cluster administrator can configure dynamic provisioners to service one or more storage classes."